### DISCUSSION TITLE
Add functionality to select cut, copy, paste options from context menu

### DISCUSSION THREAD

**Discussion by JameyEvans** [iddan/react-spreadsheet#369] [Ideas]
I recently deployed an app that utilizes react-spreadsheet to allow users to paste data from a spreadsheet into the application.  Shortly after deployment, I discovered that a lot of older users do not know how to use keyboard shortcuts to copy and paste.   It would be nice to include a context menu with these options.

**Comment by iddan** [comment]
PRs welcome
